Advisor docs (#1347)
scylladb/scylla-monitoring: docs/source/advisor/nodeCLErrors.rst

Some operation failed due to unsatisfied consistency level
----------------------------------------------------------

> ScyllaDB uses data replication, that means that a query is sent to a coordinator node which in turn sends it to the replica nodes (the nodes that hold and persist the replicated data) it then collects the replies and returns an answer. A query `Consistency Level`_, determines the number or replicas replied are needed before the coordinator returns an answer.

Reviewer: ```suggestion
ScyllaDB uses data replication, which means that a query is sent to a coordinator node sends the query to the replica nodes (the nodes that hold and persist the replicated data). The coordinator collects the replies and returns an answer. A query `Consistency Level`_, determines the number of replicas that must reply before the coordinator returns an answer.
```
